client/src/components/common/IconButton.tsx

```diff
@@ -37,8 +37,8 @@ const IconButton = styled.button<Compactable & { transparent?: boolean }>`
     color: ${(props) =>
       props.role === "primary"
         ? "var(--mi-primary-color)"
-        : "var(--mi-lighter-foreground-color)"};
-    background-color: var(--mi-icon-button-background-color);
+        : "var(--mi-black)"};
```

Comment:
Ditto this question about colors in the dark / light mode

Reply:
I'd decided to do that for the sake of simplicity since it's not really a huge difference visually in either dark or light mode but we could add an inversion?